src/collective/nitf/tests/test_views.py

```diff
@@ -114,17 +108,12 @@ def setUp(self):
         self.view = api.content.get_view(self.name, self.n1, self.request)
 
     def test_slideshow_view_render(self):
-        self.assertNotIn('<div class="slideshow-player">', self.view())
-        self.assertNotIn('<div class="slideshow-pager">', self.view())
+        self.assertNotIn('<div class="swiper-container">', self.view())
+        self.assertNotIn('<div class="swiper-pagination">', self.view())
         image = api.content.create(self.n1, 'Image', 'foo')
         set_image_field(image, FRACTAL, 'image/jpeg')
-        self.assertIn('<div class="slideshow-player">', self.view())
-        self.assertIn('<div class="slideshow-pager">', self.view())
-
-    def test_render_js_resources(self):
-        rendered = self.view()
-        for js in JS_RESOURCES:
-            self.assertIn(js, rendered)
+        self.assertIn('<div class="swiper-container">', self.view())
```

Comment:
Is this to make sure that the js was correctly loaded? Because these tags already exist on the template with js or not.

Reply:
these tests don't run javascript.  this test just make sure to include the slideshow when there is an image into the NITF object.